demos/device/package.json

```diff
@@ -3,11 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "amazon-chime-sdk-js": "file:../..",
     "@types/aws-iot-device-sdk": "^2.1.x",
     "@types/aws-sdk": "^2.7.0",
+    "amazon-chime-sdk-js": "file:../..",
+    "aws-iot-device-sdk": "^2.2.4",
```

Comment:
nit: `^2.2.3` will cover the version from 2.2.3 < 3.0.0 e.g. 2.9.999. You do not need to update this `package.json` unless you need `2.2.4`.

Reply:
NVM, I just noticed that `aws-iot-device-sdk` was updating to 2.2.4 to address security issue.